packages/server/lib/modes/run.ts

```diff
@@ -224,15 +225,31 @@ async function trashAssets (config: Cfg) {
   }
 }
 
-async function startVideoRecording (options: { previous?: VideoRecording, project: Project, spec: SpecWithRelativeRoot, videosFolder: string }): Promise<VideoRecording> {
+async function startVideoRecording (options: { previous?: VideoRecording, project: Project, spec: SpecWithRelativeRoot, videosFolder: string, overwrite: boolean }): Promise<VideoRecording> {
   if (!options.videosFolder) throw new Error('Missing videoFolder for recording')
 
-  function videoPath (suffix: string) {
-    return path.join(options.videosFolder, options.spec.relativeToCommonRoot + suffix)
+  async function videoPath (suffix: string, ext: string) {
```

Comment:
this might read a bit easier when invoked if the suffix was an optional argument and the last argument of the function.

Reply:
I have changed it, so that the suffix is second arg, with a default value, so if nothing is passed it takes `''` as default value. That said I'm not sure if this is better or not from before, so can you check once? I have added it as a separate commit, so can be directly reverted if not useful - [07577ed](https://github.com/cypress-io/cypress/pull/30673/commits/07577ed25df157e9f3d8af55df4e44c028ecad54)